### DISCUSSION TITLE
"Internal Server Error" with Mozilla Hubs Cloud Custom Client

### DISCUSSION THREAD

**Discussion by asathiya007** [mozilla/hubs#3029] [Hubs Cloud Questions] [answered]
Hey everyone! I am new to Mozilla Hubs cloud and I am trying to follow the instructions here: https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html but I am getting the following error when I run npm start: 
`Error: Error fetching Hubs Cloud config "Internal Server Error"
    at fetchAppConfigAndEnvironmentVars (/home/asathiya007/Desktop/GeorgiaTech/Academics/Fall2020/VIP/Hubs/hubs/webpack.config.js:153:11)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async module.exports (/home/asathiya007/Desktop/GeorgiaTech/Academics/Fall2020/VIP/Hubs/hubs/webpack.config.js:202:21)`
 
When I ran `npm ci` earlier, I noticed this in the logging output: `Skipping 'fsevents' build as platform linux is not supported`

I am running an Ubuntu Linux machine.

Does anyone know how I can successfully set up a custom client on Hubs Cloud?

**Comment by robinkwilson** [comment, marked as answer]
Are you basing your branch off of the hubs-cloud branch in the repo?

**Reply by asathiya007** [reply]
Yeah, I am

**Reply by robinkwilson** [reply]
Quick context questions:
1. Do you have a deployed Hubs Cloud instance?
2. Did you `npm run login` and click on the link inside your mail to login to an *admin* account? Double check that this is an admin account (data loads in the menus on your HC Admin panel).
3. Can you also try running `npm run dev`? Runs your client against our developer servers as a comparison.

Thanks!

**Reply by robinkwilson** [reply]
Marking as answered to remove from our pipeline. Please @ me if you're still having an issue!
